src/library_syscall.js

```diff
@@ -910,6 +913,10 @@ var SyscallsLibrary = {
   __syscall153: function(which, varargs) { // munlockall
     return 0;
   },
+  __syscall158: function(which, varargs) { // sched_yield
+    // always succeeds, not guaranteed to do anything anyways
+    return 0;
```

Comment:
This may be legal, but will it cause surprising behavior? Would it make more sense to #ifdef out any code that expects to use it?